src/genie/libs/parser/iosxr/show_pim.py

```diff
@@ -532,3 +534,128 @@ def cli(self, vrf='default', af='ipv4', output=None):
                 continue
 
         return parsed_dict
+
+
+# ==========================================================================
+# Schema for 'show pim topology summary'
+# ==========================================================================
+class ShowPimTopologySummarySchema(MetaParser):
+    """ Schema for show pim [vrf <vrf>] topology summary. """
+
+    schema = {
+        'vrf':
+            {Any():
+                {'active_group_ranges': int,
+                 'no_group_ranges': int,
+                 'no_g_routes': int,
+                 'no_g_rpt_routes': int,
```

Comment:
this one was ok with `no_sg_rpt_routes`

Reply:
Hi @brennobrenno , a reminder of this comment

Reply:
hi @tahigash and @xiaoxinz-cisco, changing no_g_rpt_routes to no_sg_rpt_routes might be confusing though. Since (S,G) routes is no_g_routes and (*,G) routes is no_sg_routes, someone might interpret no_sg_rpt_routes as (*,G) RPT routes which doesn't make sense. Let me know if it's okay

Reply:
ah, you got the idea as opposite, I think... if source is `*`, just `g`, if source is `S`, `sg`.
```
(*, G) routes -> no_g_routes
(S, G) routes -> no_sg_routes
(S, G) routes -> no_sg_rpt_routes
```
I think parsing code is also opposite. sorry for confusing. could you please check and update?

Reply:
Hi @tahigash, aa yep I had it backwards, made the changes now though!

Reply:
thanks for addressing my comment.